package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "arlas-d3",
+  "version": "0.0.1",
+  "description": "D3 components library for ARLAS Web App",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
+  "scripts": {
+    "build-release": "tsc -p tsconfig-build.json && mkdir dist/histograms/style && cp -r src/histograms/style/ ./dist/histograms/",
```

Comment:
A build-release script should be generic, remove histograms parts and replace by gulp script

Reply:
Gulp script added